src/index.html

```diff
@@ -30,7 +30,7 @@
     <img id="banner" src="banner.jpg" alt="Banner" height="240">
 
     <h1>
-        About to Jovian
+        Welcome to Jovian
```

Comment:
Maybe this title should be more introducing.

Reply:
Ok I am doing the change right now